README.md

```diff
@@ -162,6 +162,7 @@ This section is a great place to start if you want to get into improving OpenStr
 * [Cycle.Travel](https://cycle.travel/map) - An OSM-based bike route-planner, together with extensive editorial content about long-distance cycle routes and city cycling. ([Wiki](https://wiki.openstreetmap.org/wiki/Cycle.travel))
 * [OSM Landuse](https://osmlanduse.org) - OSM based landuse and landcover WebGIS application.
 * [OsmAPP](https://osmapp.org) - A browsable map showing data and offering simple POI-editing capabilities. ([Source Code](https://github.com/zbycz/osmapp))
+  * [openclimbing.org](https://openclimbing.org) - A map for rock climbers with editor for creating interactive climbing guides. The app is based on OsmAPP.
```

Comment:
if applicable remove indent:
```suggestion
* [openclimbing.org](https://openclimbing.org) - A map for rock climbers with editor for creating interactive climbing guides. The app is based on OsmAPP.
```

and move to end of category (below Streets GL)